packages/@aws-cdk/aws-elasticloadbalancingv2/lib/shared/base-target-group.ts

```diff
@@ -373,6 +373,6 @@ export interface LoadBalancerTargetProps {
  *     app/my-load-balancer/50dc6c495c0c9188
  */
 export function loadBalancerNameFromListenerArn(listenerArn: string) {
-    const arnParts = new cdk.FnSplit('/', listenerArn);
-    return `${new cdk.FnSelect(1, arnParts)}/${new cdk.FnSelect(2, arnParts)}/${new cdk.FnSelect(3, arnParts)}`;
+    const arnParts = cdk.Fn.split('/', listenerArn);
```

Comment:
At this stage I would kinda like it if `Fn.split` would return an object that has a `select(idx: number): string` method that would do the `Fn.select` for you.